snapcraft/internal/meta.py

```diff
@@ -207,6 +211,33 @@ def _wrap_exe(self, command, basename=None):
 
         return os.path.relpath(wrappath, self._snap_dir)
 
+    def _generate_hook(self, hook, command):
```

Comment:
Agreed, I just wanted to conform to the order already there for the binary wrappers. But reordering them as well isn't that much of a change, so I did that, too.